data/common.yaml

```diff
@@ -14,27 +14,27 @@ profile_firewall::ignores: {}
 # Leave some generic inbuilt_chains, but each OS should have their own
 # copy of inbuilt_chains in hiera under data/os/..
 profile_firewall::inbuilt_chains:
```

Comment:
I audited and fixed the ordering of these. Nothing was technically wrong with the ordering, but I sorted each group of tables alphabetically.